principles/3_Privacy.md

```diff
@@ -0,0 +1,51 @@
+# Privacy 
+
+## Data Minimization
+
+* The minimum amount of data possible shall be collected, for the purposes for which it is gathered. Data that is not used in the disclosure to healthcare authorities should not be collected unless it is essential to the purposes of the application
+* Users can choose how much data to share from symptom surveys
+
+## Consent
+* A member of the public (MoP) shall consent to data being collected or transmitted explicitly, that is it should not be implied by usage, or mixed with another consent
```

Comment:
Suggested edit for clarity: "A member of the public (MoP) shall be explicitly asked for consent to data being collected or transmitted. Consent should not be implied by usage, or mixed with any other consent requests."

Reply:
done